packages/@aws-cdk/aws-s3-deployment/README.md

```diff
@@ -325,6 +325,20 @@ The value in `topic.topicArn` is a deploy-time value. It only gets resolved
 during deployment by placing a marker in the generated source file and
 substituting it when its deployed to the destination with the actual value.
 
+## Keep Files Zipped
+
+By default, files are zipped, then extracted into the destination bucket.
+You can use the option `extract: false` to disable this behavior, in which case, files will remain in a zip file when deployed to S3.
```

Comment:
Say something about the target filename and how it can be acquired by users.

Reply:
I think by target filename you are referring to the `*.zip` file that is found in the target S3 bucket after bucket deployment. Is that correct?

Reply:
Yes.